docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -122,6 +122,17 @@ automatically joined as required:
  * `MyRecordSubclass_Live` table: Contains only live data for subclass columns
  * `MyRecordSubclass_Versions` table: Contains only version history for subclass columns
 
+While not explicit DataObjects, `many_many` relationships create their own sets of records on their own tables.
```

Comment:
That's not entirely true; `many_many` supports `through` syntax which does use an explicit DataObject. You should make the distinction here and describe how it differs from the traditional syntax with auto-scaffolded table.